openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -115,149 +85,287 @@
+        the project scope. Users and gruops can be associated with, or bound
+        to, multiple roles at the same time.
+
+        For service accounts that might also have privileged access, Security
+        Context Constraints (SCC) should be used to limit privileged actions
+        in service accounts.
+
+        For the cluster, the kubeadmin user should be completely disabled.
+
+        More work is being done to address this control, Engineering
+        progress can be tracked via:
+
+        https://jira.coreos.com/browse/CMP-84
 
     - key: b
-      text: |
-        'This is an organizational control and not applicble to the configuration
-        of OpenShift.'
+      text: |-
+        This is an organizational control and not applicble to the configuration
+        of OpenShift.
     - key: c
-      text: |
-        'This is an organizational control and not applicble to the configuration
-        of OpenShift.'
+      text: |-
+        This is an organizational control and not applicble to the configuration
+        of OpenShift.
+
+- control_key: AC-2 (8)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |-
+        This control is applicable to the identity service provider,
+        and outside the scope of OpenShift configuration.
 
 - control_key: AC-2 (9)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: |
-        'This control is applicable to the identity service provider,
-        and outside the scope of OpenShift configuration.'
+    - text: |-
+        This control is applicable to the identity service provider
+        and outside the scope of CoreOS configuration.
 
 - control_key: AC-2 (10)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: |
-        'OpenShift does not have the concept of group accounts.
+    - text: |-
+        OpenShift does not have the concept of group accounts.
 
         Should the organization allow shared credentials, the
         creation and management of such would fall under the scope
-        of operating the shared identity service and not OpenShift.'
+        of operating the shared identity service and not OpenShift.
 
 - control_key: AC-2 (11)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: not applicable
+  implementation_status: partial
   narrative:
-    - text: |
-        'This control is applicable to the identity service provider,
-        and outside the scope of OpenShift configuration.'
+    - text: |-
+        This control will only be met as partial when OpenShift is
+        configured to connect to an Identity Provider (IdP).
+
+        More work is being done to address this control, Engineering
+        progress can be tracked via:
+
+        https://jira.coreos.com/browse/CMP-210
```

Comment:
How can this be not applicable but also require engineering?

Reply:
@shawndwells control is now 'partially' applicable.